packages/apollos-church-api/src/data/content-item-slugs/index.js

```diff
@@ -0,0 +1,5 @@
+import schema from './schema';
+import resolver from './resolver';
```

Comment:
could you import from packages/apollos-church-api/src/data/content-items/resolver.js instead of using a dummy file?